core/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -165,7 +182,7 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
   @tailrec
   final def findFirst(start: FormatToken, end: Token)(
       f: FormatToken => Boolean): Option[FormatToken] = {
-    if (start.left.start < end.start) None
```

Comment:
Whoah, good catch! Was this method used somewhere before? Logic bugs like this tend to pop up in dead code.